deploy/nl/embeddings.yaml

```diff
@@ -27,6 +27,10 @@ indexes:
     store: MEMORY
     embeddings: gs://datcom-nl-models/embeddings_undata_2024_03_20_11_01_12.ft_final_v20230717230459.all-MiniLM-L6-v2.csv
     model: ft-final-v20230717230459-all-MiniLM-L6-v2
+  undata_ilo_ft:
```

Comment:
As validation, should we have an e2e test like https://github.com/datacommonsorg/website/blob/f7433df7f18fb0b306b7d04b101fe4701dfcf5eb/server/integration_tests/explore_test.py#L278-L281 ?

(And later when the data has been added to main DC, we can [do fulfill as well](https://github.com/datacommonsorg/website/blob/f7433df7f18fb0b306b7d04b101fe4701dfcf5eb/server/integration_tests/explore_test.py#L587-L590))

Reply:
Added